conf/repo/maintainer.yml

```diff
@@ -1,14 +1,16 @@
 maintainer-quality-tools:
-  branches: []
+  branches:
+    - "18.0"
   category: Maintainer
-  default_branch: master
+  default_branch: "18.0"
```

Comment:
Careful with these

Reply:
Yep, I've seen it. I'll handle some blacklisting :)

Reply:
Fixed